### PR TITLE
completed create event and delete an event

### LINKED ISSUE #127: As an admin, I can create an event

### PR DESCRIPTION
**Pull request by bhargavisatpathy**
Now an Admin can create an event closes #127 

Admin can delete an event #129

Updated seed data for a default image for event.

### CODE REVIEW

File: app/helpers/admin/images_helper.rb [hunk 1]
```diff
@@ -10,12 +10,6 @@ def update_image(param_path)
   end
 
   def add_image(param_path)
```
Comment (nathanows) [inline app/helpers/admin/images_helper.rb:12]: I think I ended up needing to write this method all funky like this with two weird methods for the edit functionality later on, but I could be completely wrong. either way, much cleaner now. nice work! :sparkles:

File: app/views/admin/events/_form.html.erb [hunk 2]
```diff
@@ -1,6 +1,12 @@
 <%= bootstrap_form_for [:admin, @event], :html => { multipart: true }  do |f| %>
   <%= f.text_field :title %>
   <%= f.text_area :description %>
+  <%= f.collection_select(:venue,
```
Comment (nathanows) [inline app/views/admin/events/_form.html.erb:4]: Aren't these collection_select's fun :tada:?! I spent way too long on these today :(.

File: app/controllers/admin/events_controller.rb [hunk 3]
```diff
@@ -7,6 +7,19 @@ def index
     @events = Event.order(updated_at: :desc)
   end
 
+  def new
+    @event = Event.new
+  end
+
+  def create
```
Comment (nathanows) [inline app/controllers/admin/events_controller.rb:14]: looks good, but how does this handle error messages? 

I've been using the format:
def create
    all the event stuff except save
    if (at)event.save
        redirect_to admin_events_path
    else
        render 'form'
    end
end

which allows for the form to regenerate with error messages and all of the content that had been typed in. Is that working anyway?

File: app/controllers/admin/events_controller.rb [hunk 4]
```diff
@@ -7,6 +7,23 @@ def index
...
+    @event = Event.new(event_params)
+    @event.image = add_image(params[:event][:images])
+    @event.category = Category.find(params[:event][:category])
+    @event.venue = Venue.find(params[:event][:venue])
+    if @event.save
+      redirect_to admin_events_path
+    else
+      flash[:errors] = @event.errors.full_messages.uniq.join("<br>")
```
Comment (scottcrawford03) [inline app/controllers/admin/events_controller.rb:22]: glad you snagged this line of code. works well. :large_blue_circle:

**Comment by scottcrawford03** [comment]
looks good. the error messages have been addressed. well done! :chart_with_upwards_trend:

**Pull request merged**